README.asciidoc

```diff
@@ -0,0 +1,112 @@
+This is the documentation file for team TWELVE.
+Team members and their main roles are:
+
+	Pille Pärnalaas - Programming
+	Kenno Kallastu - Mechanics
+	Stepan Denysenko - Electronics
+
+
+DAILY BLOG
```

Comment:
It would be a bit better to use section title syntax.

This might be useful:
https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#section-titles